### PR TITLE
Add Daniel Marelius Bjørnstad's profile page

### PR DESCRIPTION
**Pull request by lastis**

### CODE REVIEW

File: content/about/employees/daniel/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+development. With expertise in multiple stages of project development, Daniel’s work spans from the initial definition
+of project goals and value propositions to the deployment of final products, including services, dashboards, reports,
+production models, and APIs. His background in neuroscience provides a strong foundation in complex data processing and
+analysis, allowing him to tackle challenging problems in diverse domains. His professional experience started in 2016.
+<!--more-->
+Throughout his career, Daniel has been involved in diverse projects, such as designing audio analysis systems for
+machinery maintenance, where he executed key tasks like feature extraction, data modeling, and cloud infrastructure
+deployment. Additionally, he played an important role in optimizing production efficiency for a gypsum factory, where
+his efforts in data analysis and machine learning were pivotal. He also worked on real estate socio-economic platform
+which showcased his ability to integrate ETL processes with the development of a full-stack system.
```
Comment (alexanfl) [inline content/about/employees/daniel/index.md:25]: Some suggestions for the flow and sound of the text. Pick and choose which suggestions you want to incorporate. I also think you should mention the Telia project here.

```suggestion
Daniel is a seasoned software developer with a PhD in neuroscience, specializing in data analysis and cloud software
development. With expertise in multiple stages of project development, Daniel’s work spans from the initial definition
of project goals and value propositions to the deployment of final products, including services, dashboards, reports,
production models, and APIs. His background in neuroscience provides a strong foundation in complex data processing and
analysis, allowing him to tackle challenging problems in diverse domains. He started working professionally with software development in 2016.
<!--more-->
Throughout his career, Daniel has been involved in several projects. He has worked on designing audio analysis systems for
machinery maintenance, where he executed key tasks like feature extraction, data modeling, and cloud infrastructure
deployment. Additionally, he played an important role in optimizing production efficiency for a gypsum factory, where
his data analysis and machine learning efforts were pivotal. Before his PhD, he also worked on a real estate socio-economic platform. This showcased his ability to integrate ETL processes with developing a full-stack system.
```

**Comment by alexanfl** [comment]
Great work! It looks overall very good. I tried to shorten som sentences to make it easier to read some places, but see which changes you like, etc. Also think the Telia project should be mentioned, unless it was there and I didn't understand :sweat_smile:

**Comment by lastis** [comment]
Ready for review again

**Review by alexanfl — APPROVED** [review]
lgtm!

**Pull request merged**